aws/data_source_aws_vpc_peering_connections_test.go

```diff
@@ -0,0 +1,83 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDataSourceAwsVpcPeeringConnections_basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceAwsVpcPeeringConnectionsConfig,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.aws_vpc_peering_connections.test_by_filters", "ids.#", "2"),
+				),
+				ExpectNonEmptyPlan: true,
```

Comment:
This acceptance testing should successfully complete without this line:

```suggestion
```

If you have questions about how to get this working, please reach out.